### ISSUE TITLE
 Video playback freeze(only picture) when player downloads new chunks, audio continue...

### ISSUE THREAD

**Issue by ferrys21** [mangui/flashls#482]
Hi, We tried to use your project for HLS streaming, but our live streams have problems. By playing video the picture freeze suddenly - only picture and audio continue... after some seconds continue video further... I have investigated, that the gap in video(picture) is created, when player is downloading new chunks... All buffers are ok... We have quite fast connection - about 100 Mbit/s... please try this stream... http://live.dakr-tv.cz/...  VLC and OSMF HLS plugin dont have problem with this and all other our streams... Problem has occured with all players and all examples... THX L.F.

**Comment by mangui** [comment]
I can observe some glitch at playback start :
on first fragment parsing we could see that it does not start with an I-Frame. also PMT is not located at the beginning of the file.
looks like potential segmentation issue.

**Comment by ferrys21** [comment]
I dont think so.... The stream plays fluently... Synced iframe is necessary in multibitrate hls stream. The problem has never been observed in vlc, apple devices, html5 players... Dont worry about it, we bought osmf hls plugin and looking good. The picture freezes, when player starts download new files... after downloading in memory the picture continue... Better network speed = more visible freezing. thx
